templates/CRM/Activity/Form/Selector.tpl

```diff
@@ -67,7 +67,7 @@
     <td>
     {if $row.mailingId}
       <a href="{$row.mailingId}" title="{ts}View Mailing Report{/ts}">{$row.recipients}</a>
-    {elseif $row.recipients}
+    {elseif !empty($row.recipients)}
```

Comment:
I think this one might be an issue with the hardening @eileenmcnaughton is doing.

Reply:
yeah - you can't add empty to tpls (or isset) - you need to make sure it is assigned or you are just moving the problem down the road. I've been taking out the empties

Reply:
Ok, I thought it was only the isset that had to go